basenko_denis/src/components/MessageField/MessageField.jsx

```diff
@@ -0,0 +1,76 @@
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import './messagesField.css'
+
+import React from 'react';
+
+import { MessagesList } from 'components/MessagesList/MessagesList';
+
+export class MessageField extends React.Component {
+    
+    state ={
+        messages:[
+            {author:'bot',text:'Привет я бот Валера'},
+            {author:'bot',text:'Я отвечу на любой твой вопрос.'},
+            {author:'bot',text:'Задавай?'},
+        ],
+        newMessage: '',
+        author:''
+    }
+    keyDownEnter = (event) => {
+        if(event.ctrlKey && event.keyCode === 13){
+            this.onSentMessage(event)
+        }
+    }
+    onChangeName = (event) =>{
+        this.setState({author: event.target.value})
+    }
+    answerForBot = () =>{
+        this.setState({
+            messages:[...this.state.messages, 
+                        {author:'bot', 
+                         text:this.state.author +', мая-твая нипонимать!'}
+                    ],
+        })
+    }
+    onSentMessage = (event) =>{
```

Comment:
Не забывайте об оформлении кода, читаемость кода - очень важно
```suggestion
    onSentMessage = (event) => {
```